apps/pie-microsite/src/content/pages/foundations/spacing/alias/alias.md

```diff
@@ -0,0 +1,13 @@
+---
+eleventyNavigation:
+    key: Alias tokens
+    parent: Spacing
+    order: 3
+---
+## Static spacing
+Static spacing tokens are used in instances where spacing inside, between or around components needs to be the same regardless of the the size of the viewport.
+
+{% tokensTable {
+ path: 'theme.jet.spacing.alias',
+ prefix: 'spacing'
+} %}
```

Comment:
I think we should remove this until we pick up the ticket(s) to properly fill out this page